packages/components/src/components/table/Table.tsx

```diff
@@ -0,0 +1,191 @@
+import * as React from 'react';
+import { Checkbox, Icon, Dropdown } from '..';
+
+export type TableProps = {
+  items: { [key: string]: string }[];
+  header: { [key: string]: string }[];
+  rowsPerPage?: number;
+  showSorting?: boolean;
+  showPagination?: boolean;
+  showCheckbox?: boolean;
+};
+
+export const Table: React.FC<TableProps> = ({
+  items,
+  header,
+  showCheckbox,
+  rowsPerPage,
+  showSorting,
+  showPagination,
+}: TableProps) => {
+  const [data, setData] = React.useState(items);
+  const [currentPage, setCurrentPage] = React.useState(1);
+  const [select, setSelect] = React.useState(rowsPerPage);
+  const [order, setOrder] = React.useState('DSC');
+  const [headingKey, setHeadingKey] = React.useState({
+    company: true,
+    contact: true,
+    country: true,
+  });
+
+  const renderIcon = (key) => {
+    if ([key] && headingKey[key]) {
+      return <Icon iconName="drop-down" onClick={() => sortingColumn(key)} />;
+    } else {
+      return <Icon iconName="drop-up" onClick={() => sortingColumn(key)} />;
+    }
+  };
+
+  const sortingColumn = (key) => {
+    if (key && order === 'ASC') {
```

Comment:
small improvement if you don't want to repeat key 
you could add a line before 
if(!key) return;
then focus only on the order